From 29d793b2fc115cb5956d0a4ef42d18b652231f04 Mon Sep 17 00:00:00 2001
From: Peter Eisenmann <p3732@getgoogleoff.me>
Date: Tue, 23 Sep 2025 17:50:54 +0200
Subject: [PATCH] app-chooser: Adjust to GTK 4.20's list view

(cherry picked from commit fee7ee6ece7a3eb4e4001edbd64d06abefe7ad6f)
---
 src/nautilus-app-chooser.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/nautilus-app-chooser.c b/src/nautilus-app-chooser.c
index a14d550250..4f1c32edb7 100644
--- a/src/nautilus-app-chooser.c
+++ b/src/nautilus-app-chooser.c
@@ -131,7 +131,7 @@ focus_app_chooser_widget (NautilusAppChooser *self)
     g_return_if_fail (GTK_IS_SCROLLED_WINDOW (child));
 
     child = gtk_widget_get_first_child (child);
-    g_return_if_fail (GTK_IS_TREE_VIEW (child));
+    g_return_if_fail (GTK_IS_TREE_VIEW (child) || GTK_IS_LIST_VIEW (child));
 
     gtk_widget_grab_focus (child);
 
-- 
GitLab

